### PR TITLE
Update dependencies

### LINKED ISSUE #3680: Update dependencies to use `ubyte`
Update dependencies required to use `ubyte`.

### PR DESCRIPTION
**Pull request by mariaschett**
Fixes #3680 .

### CODE REVIEW

File: oak_functions_client/src/rekor.rs [hunk 1]
```diff
@@ -18,10 +18,11 @@
 //! verifying signatures in a Rekor LogEntry.
 
 use anyhow::Context;
-use ecdsa::Signature;
+
```
Comment (tiziano88) [inline oak_functions_client/src/rekor.rs:21]: extra blank line

File: oak_functions_client/src/rekor.rs [hunk 2]
```diff
@@ -188,8 +189,9 @@ pub fn verify_rekor_log_entry(
     let entry = parsed.values().next().context("no entry in the map")?;
 
     // Parse base64-encoded entry.body into an instance of Body.
-    let body_bytes =
-        base64::decode(entry.body.clone()).context("couldn't decode Base64 signature")?;
+    let body_bytes: Vec<u8> = general_purpose::STANDARD
```
Comment (tiziano88) [inline oak_functions_client/src/rekor.rs:192]: perhaps importing `base64::prelude::BASE64_STANDARD` at the top would make the code here more readable (it was unclear to me what `general_purpose::STANDARD` actually refers to when I first skimmed through).

**Review by tiziano88 — APPROVED** [review]
Thanks for the project-wide upgrade

File: oak_functions_client/src/rekor.rs (hunk 2 above)
Comment (mariaschett) [reply oak_functions_client/src/rekor.rs]: Done! (`general_purpose::STANDARD` seems reasonable for general purpose and standard things which I presumed we want to use)

**Review by mariaschett — COMMENTED** [review]
Great, thanks for the review!

Oak nearly has updated dependencies now: "s2" from in the Weather Exampleis not and some dependency in Oak Restricted Kernel.

A follow up related issue: https://github.com/project-oak/oak/issues/3683 , which should also help us removing dependencies.

**Pull request merged**